modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -208,11 +209,50 @@ export async function sendShareToBitgo(
       }
       const sShare = share as SShare;
       signatureShare = convertSDShare({ sShare, dShare });
-      const signatureRecord = await sendSignatureShare(bitgo, walletId, txRequestId, signatureShare);
-      responseFromBitgo = parseSignatureShare(signatureRecord);
+      await sendSignatureShare(bitgo, walletId, txRequestId, signatureShare);
+      responseFromBitgo = sShare;
```

Comment:
I think we should get a fully signed signature from bitgo at this point via get Tx request?

Reply:
tx request will be signed and wp cleans up signature shares. at this point the share don't matter.

Reply:
Probably good if the client get the full signature as a response as well. Think we already is calling a getTxRequest as last step in the sign function, probably only need to move here may be

Reply:
Yep I agree, the user gets signed tx however not that shares